### ISSUE TITLE
How to predict landmark for new images?

### ISSUE THREAD

**Issue by shzhjxc** [svip-lab/HRNet-for-Fashion-Landmark-Estimation.PyTorch#7]
First, I would like to thank authors for sharing the code. The work inspired me a lot and I‘m trying to followup your method for finding keypoints using my own input images. 
However, I found that it is hard to directly modify test.py to meet a situation when the input is not the validation set, which really confused me. Is there a straightforward way to generate landmark with input images not in the deepfashion2 dataset?

**Comment by ShenhanQian** [comment]
Currently, our code doesn't support inference beyond a dataset. You might need to re-define the dataset class to achieve this.

Also, this model is supposed to work after an object detector that can predict bounding boxes and categories first.
